app/models/labware_creators/donor_pooling_plate.rb

```diff
@@ -145,15 +122,14 @@ def barcodes=(barcodes)
       @minimal_barcodes = barcodes.compact_blank.map(&:strip)
     end
 
-    # Returns the number of pools based on the sample count from the lookup
-    # table.
+    # Returns the number of pools that this group of wells should be split between, pulled from request metadata.
     #
-    # @return [Integer] The number of pools.
-    def calculated_number_of_pools
-      return if source_wells_for_pooling.blank?
-
-      # div enfoces integer division
-      source_wells_for_pooling.count.div(number_of_samples_per_pool)
+    # @param [Array<Well>] group A group of wells from the source plate.
```

Comment:
Note: a group contains wells from the same study-project. number_of_pools is specified per study-project.

Another note: technically, the labware creator assumes more than one source plate but it is not relevant in this context.